Telegraf: fix Windows install instructions. (#5086)
influxdata/docs-v2: content/telegraf/v1.26/install.md

brew install telegraf
```

> Run one of the following commands to start Telegraf collecting and processing metrics:

Reviewer: ```suggestion
Run one of the following commands to start Telegraf and begin collecting and processing metrics:
```

To have launchd start Telegraf as a service after the next login:

```
ln -sfv /usr/local/opt/telegraf/*.plist ~/Library/LaunchAgents
```

> To start Telegraf as a service now:

Reviewer: ```suggestion
Start Telegraf as a launchd service:
```

    and extract its contents to `C:\Program Files\InfluxData\telegraf\`:

    ```powershell
    wget `
    https://dl.influxdata.com/telegraf/releases/telegraf-{{% latest-patch %}}_windows_amd64.zip `
    -UseBasicParsing `
    -OutFile telegraf-{{< latest-patch >}}_windows_amd64.zip `
    Expand-Archive .\telegraf-{{% latest-patch %}}_windows_amd64.zip `
    -DestinationPath 'C:\Program Files\InfluxData\telegraf\'
    ```

2.  Choose one of the following steps to place your `telegraf.exe` and `telegraf.conf` files in `C:\Program Files\InfluxData\telegraf`:

    - Move the `telegraf.exe` and `telegraf.conf` files from
    `C:\Program Files\InfluxData\telegraf\telegraf-{{% latest-patch %}}` to the parent directory `C:\Program Files\InfluxData\telegraf`--for example:

        ```powershell
        cd "C:\Program Files\InfluxData\telegraf";
        mv .\telegraf-{{% latest-patch %}}\telegraf.* .
        ```

    - **Or**, create a [Windows symbolic link (Symlink)](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/) for
    `C:\Program Files\InfluxData\telegraf` that points to the extracted directory.

>     > The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in `C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.

Reviewer: ```suggestion
    {{% note %}}
The remaining instructions assume that `telegraf.exe` and `telegraf.conf` files are stored in
`C:\Program Files\InfluxData\telegraf` or that you created a Symlink to point to this directory.
    {{% /note %}}
```

> 3.  Optional: Disable the [`inputs.processes` plugin](/{{% latest "telegraf" %}}/plugins/#input-processes).

Reviewer: ```suggestion
3.  _(Optional)_ Disable the [`inputs.processes` plugin](/telegraf/v1.26/plugins/#input-processes).
```

Author: https://developers.google.com/style/procedures#optional-steps

    This plugin doesn't support Windows and returns an error when run with the `--test` flag.
    Open `telegraf.conf` in your editor and comment the `inputs.processes` configuration lines.

    ```toml
    ...
    # This plugin ONLY supports non-Windows
    # [[inputs.processes]]
    ...
    #  # use_sudo = false
    ...
    ```

> 4.  Optional: Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/{{% latest "telegraf" %}}/plugins/#input-win_services) configuration line:

Reviewer: ```suggestion
4.  _(Optional)_ Enable a plugin to collect Windows-specific metrics--for example, uncomment the [`inputs.win_services`  plugin](/telegraf/v1.26/plugins/#input-win_services) configuration line:
```
